### PR TITLE
chore: add CDK Construction Zone episodes list to main README

### PR DESCRIPTION
**Pull request by NetaNir**
While we wait on the AWS twitch channel maintainers to create a playlist for the first season, added a list with the episodes 

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,14 @@ this capability, please see the
 
 ## More Resources
 * [CDK Workshop](https://cdkworkshop.com/)
+* **CDK Construction Zone** - A Twitch live coding series hosted by the CDK team, season one episodes:
+  * [S1E1](https://www.twitch.tv/videos/917691798):cdk triggers part 1
```
Comment (eladb) [inline README.md:158]:
```suggestion
  * [S1E1](https://www.twitch.tv/videos/917691798): triggers part 1
```

File: README.md [hunk 2]
```diff
@@ -154,6 +154,14 @@ this capability, please see the
...
+  * [S1E2](https://www.twitch.tv/videos/925801382): cdk triggers part 2
+  * [S1E3](https://www.twitch.tv/videos/944565768): cdk triggers part 3 (final) 
+  * [S1E4](https://www.twitch.tv/aws/video/960287598): cdk tokens
```
Comment (eladb) [inline README.md:161]:
```suggestion
  * [S1E4](https://www.twitch.tv/aws/video/960287598): Tokens Deep Dive
```

**Review by eladb — COMMENTED** [review]
1. Change all to Title Case
2. Remove "CDK" prefix from titles

(Do we want to add list of participants?)

**Review by eladb — APPROVED** [review]

**Comment by NetaNir** [comment]
Thanks @eladb, I wanted to minimize the space this takes in the main readme, so I didn't add any details such as panelists. Let me know if you think it's appropriate and I'll add it.

**Comment by eladb** [comment]
I think some information can be useful for search engines and humans to be able to tell if the episode is relevant. I wouldn't over do it.

**Pull request merged**